composeApp/src/commonTest/kotlin/com/arkivanov/minesweeper/game/DefaultGameComponentTest.kt

```diff
@@ -56,21 +56,39 @@ class DefaultGameComponentTest {
 
     @Test
     fun GIVEN_game_started_WHEN_click_on_mine_cell_THEN_timer_STOPS() {
-        clickCellPrimary(x = 0, y = 0)
+        clickCellPrimary(0 by 0)
 
         coroutineScheduler.advanceTimeBy(10.seconds)
         coroutineScheduler.runCurrent()
 
-        clickCellPrimary(x = 10, y = 10)
+        val (mineLocation) = state.grid.entries.first { (_, cell) -> cell.value.isMine }
+        clickCellPrimary(mineLocation)
 
-        coroutineScheduler.advanceTimeBy(10.seconds)
+        coroutineScheduler.advanceTimeBy(20.seconds)
         coroutineScheduler.runCurrent()
 
         assertEquals(10, state.timer)
     }
 
-    private fun clickCellPrimary(x: Int, y: Int) {
-        gameComponent.onCellTouchedPrimary(x = x, y = y)
-        gameComponent.onCellReleased(x = y, y = y)
+    @Test
+    fun GIVEN_game_started_WHEN_status_win_THEN_timer_STOPS() {
```

Comment:
Sorry, last bit - it would be better to write "WHEN_win_THEN". Would it be possible to fix?

Reply:
sure!

Reply:
Thanks a lot for big help with implementing this feature  🍰